### DISCUSSION TITLE
Fastlane Json

### DISCUSSION THREAD

**Discussion by MartinGonzalez** [fastlane/fastlane#16894] [Show and tell]
I searched in fastlane plugins if there were a plugin that handle jsons and found only `load_json` and `parse_json` so I created a plugin that will centralise everything related to json handling. You can found it here [fastlane-plugin-json](https://github.com/MartinGonzalez/fastlane-plugin-json) and feel free to use it and add issues/contribute too.

Right now has:
* read_json
* write_json
* download_json
* merge_jsons

Future release will have:
* merge_to_json (having two or more hashes, merge them into a json file)

If you found it useful let me know! And if you would like to see an action let me know too :D

**Comment by joshdholtz** [comment]
@MartinGonzalez This is great! Thank you so much for sharing 😊 Your plugin will appear on your “Available Plugins” page the next time we release a new version of _fastlane_ - http://docs.fastlane.tools/plugins/available-plugins/
